lib/oidcstrategy.js

```diff
@@ -159,6 +203,10 @@ function Strategy(options, verify) {
   // validator will throw exception if a required option is missing
   var validator = new Validator(validatorConfiguration);
   validator.validate(itemsToValidate);
+
```

Comment:
Added issue for allowing http by default: https://github.com/AzureAD/passport-azure-ad/issues/164